2200017/문제3.py

```diff
@@ -0,0 +1,10 @@
+# 10505
+T = int(input()) # 테스트 케이스 수
+for t in range(1, T+1):
+    cnt = 0
+    N = int(input())
+    money = list(map(int, input().split()))
+    for i in money:
+        if i <= sum(money)/len(money):
+            cnt += 1
```

Comment:
```suggestion
    avg = sum(money) / len(money)
    for i in money:
        if avg >= i:
            cnt += 1
```
평균 값을 `for`문 밖에서 한 번만 계산하는 방법도 있습니다 😄